projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/integration-card-header.tsx

```diff
@@ -105,15 +106,29 @@ const IntegrationCardHeader = ( {
 		>
 			<div className="integration-card__header-content">
 				<div className="integration-card__header-main">
-					<div className="integration-card__service-icon-container">
-						<Icon
-							icon={ icon }
-							className={ `integration-card__service-icon ${
-								cardData.slug ? `integration-card__service-icon--${ cardData.slug }` : ''
-							}` }
-							size={ 30 }
-						/>
-					</div>
+					{ showIntegrationIcons !== false && (
+						<div className="integration-card__service-icon-container">
+							{ cardData?.iconUrl ? (
+								<img
+									src={ cardData.iconUrl as string }
+									alt={ title }
+									width={ 30 }
+									height={ 30 }
+									className={ `integration-card__service-icon ${
+										cardData.slug ? `integration-card__service-icon--${ cardData.slug }` : ''
+									}` }
+								/>
+							) : (
+								<Icon
+									icon="admin-plugins"
```

Comment:
Let's use [WP icon library](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library) instead of [Dashicons](https://developer.wordpress.org/resource/dashicons/)?

See https://wordpress.github.io/gutenberg/?path=/docs/components-icon--docs#with%20a%20dashicon

```js
import { plugins } from '@wordpress/icons';
<Icon icon={ plugins } />
```

Otherwise, we should add Dashicons somewhere as a script dependency, as we can't really predict safely when it's loaded and when not.

Reply:
Yep, this has been updated!